### PR TITLE
Update Friday Meeting Cadence

### PR DESCRIPTION
**Pull request by mcleo-d**
## Description

This PR changes the CFI meeting cadence to Friday @ 10am ET / 3pm UK as a result of the following project poll ...
- https://github.com/finos/compliant-financial-infrastructure/discussions/154#discussioncomment-1509589

### Markdown Change

- 88da9c9c1bf4bddd9033b0a96fd8916c75df5e28

```
Friday DD MMM yyyy - 10am ET / 3pm UK
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Meeting.md [hunk 1]
```diff
@@ -51,4 +51,4 @@ https://finos.webex.com/finos/j.php?MTID=m4b1d85127d1f3ca179545d6bd3291975
 
 **Project Wiki Page:** https://finosfoundation.atlassian.net/wiki/spaces/FDX/pages/904626436/Cloud+Service+Certification+Project 
 
-**Mailing List:** Email fdx-cloud-service-certification+subscribe@finos.org to subscribe to our mailing list
\ No newline at end of file
+**Mailing List:** Email fdx-cloud-service-certification+subscribe@finos.org to subscribe to our mailing list
```
Comment (abdullahgarcia) [inline .github/ISSUE_TEMPLATE/Meeting.md:54]: Are we keeping this email given the latest changes?

Comment (mcleo-d) [reply .github/ISSUE_TEMPLATE/Meeting.md:54]: Hi @abdullahgarcia - That's a great question. I'll ask @maoo how easy this is to change in Google Groups 👍🏻

Comment (mcleo-d) [reply .github/ISSUE_TEMPLATE/Meeting.md]: Hi @abdullahgarcia - The previous CSC email addresses have been updated in `README.md` and `.github/ISSUE_TEMPLATE/Meeting.md` in the commits below ...

- 5253024
- f6018f0

Many thanks,

James 🚀

**Comment by mcleo-d** [comment]
@abdullahgarcia and @peter-thomas-db - I have merged this pull request given the low risk to executable code verses the confusion caused by having incorrect information in the project documentation. Please feel free to feed back.

**Pull request merged**